[DOC] Improve the BPMN support how-to (#2321)
process-analytics/bpmn-visualization-js: docs/contributors/bpmn-support-how-to.md

- [ionicons](https://ionic.io/ionicons)
- [iconify](https://iconify.design/)
- [Noun Project](https://thenounproject.com)
- [Octicons](https://primer.style/octicons/)
- ......

[draw.io](https://github.com/jgraph/drawio) provides [BPMN shapes](https://github.com/jgraph/drawio/blob/v20.5.0/src/main/webapp/shapes/bpmn/mxBpmnShape2.js) that can be directly included in `bpmn-visualization` by adapting the JavaScript code into Typescript.
It also includes [mxGraph stencils](https://github.com/jgraph/drawio/blob/v20.5.0/src/main/webapp/stencils/bpmn.xml) that can be reused. In this case, they have to be transformed in to TypeScript code. See below for more information.

### BPMN icon and marker tips

The icon and markers of the BPMN elements must be defined in the mxGraph custom shapes and this currently must be done using
`TypeScript` code. The `IconPainter` class manages all icons rendering/painting.  

It is possible to adapt an SVG icon or an `mxGraph` stencil thanks to [mxgraph-svg2shape](https://github.com/process-analytics/mxgraph-svg2shape),

> a Java tool that will let you transform your SVG or `mxGraph` stencil file into a set of `TypeScript` commands.

Reviewer: ```suggestion
a Java tool that will let you to transform this content (in a file) into a set of `TypeScript` commands.
```
